VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -811,6 +799,15 @@ def get_workload_running(self):
         
     def set_pre_post_enabled(self):
         self.pre_post_enabled = True
+
+    def command_output_from_subprocess(self , args, process_wait_time):
+        p = subprocess.Popen(args, stdout=subprocess.PIPE)
+        while(process_wait_time > 0 and p.poll() is None):
```

Comment:
where are we setting process_wait_time?

Reply:
It's being set we call the function. We are passing it as a parameter